classes/components/forms/publication/PublishForm.inc.php

```diff
@@ -28,18 +28,23 @@ class PublishForm extends FormComponent {
 	/** @var Publication The publication being published */
 	public $publication;
 
+	/** @var \Context */
+	public $submissionContext;
+
 	/**
 	 * Constructor
 	 *
 	 * @param $action string URL to submit the form to
 	 * @param $publication Publication The publication to change settings for
+	 * @param $submissionContext \Context journal or press
 	 * @param $requirementErrors array A list of pre-publication requirements that are not met.
 	 */
-	public function __construct($action, $publication, $requirementErrors) {
+	public function __construct($action, $publication, $submissionContext, $requirementErrors) {
 		$this->action = $action;
 		$this->successMessage = __('publication.publish.success');
 		$this->errors = $requirementErrors;
 		$this->publication = $publication;
+		$this->submissionContext = $submissionContext;
```

Comment:
Is this used?

Reply:
Not in OMP PublishForm. I thought to leave it the same as in OJS, e.g., to use in hooks. Remove it?

Reply:
OK, fair enough, let's keep it in.